test/parallel/test-http-malformed-request.js

```diff
@@ -50,7 +50,4 @@ server.on('listening', function() {
     c.end();
   });
 });
-
-process.on('exit', function() {
-  assert.strictEqual(nrequests_expected, nrequests_completed);
```

Comment:
Should you do away with the assertion here, or I'm I missing something?

Reply:
There is actually no need for process.on('exit') now.

Reply:
But you still need the assertion, right?
```javascript
assert.strictEqual(nrequests_expected, nrequests_completed);
```